<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="videoLable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>241</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="options">
    <item>
     <widget class="QPushButton" name="bt_start">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{color:white;
border-image:url(./ui/resources/HomePageBtnBg.png);
font-family:微软雅黑;
font-size:16px}
.QPushButton:hover{color:#79d9ff;
border-image:url(./ui/resources/HomePageBtnBg_hover.png);
font-size:16px}</string>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_records">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{color:white;
border-image:url(./ui/resources/HomePageBtnBg.png);
font-family:微软雅黑;
font-size:16px}
.QPushButton:hover{color:#79d9ff;
border-image:url(./ui/resources/HomePageBtnBg_hover.png);
font-size:16px}</string>
      </property>
      <property name="text">
       <string>记录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_setting">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{color:white;
border-image:url(./ui/resources/HomePageBtnBg.png);
font-family:微软雅黑;
font-size:16px}
.QPushButton:hover{color:#79d9ff;
border-image:url(./ui/resources/HomePageBtnBg_hover.png);
font-size:16px}</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bt_quit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">.QPushButton{color:white;
border-image:url(./ui/resources/HomePageBtnBg.png);
font-family:微软雅黑;
font-size:16px}
.QPushButton:hover{color:#79d9ff;
border-image:url(./ui/resources/HomePageBtnBg_hover.png);
font-size:16px}</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>662</width>
     <height>129</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{background:url(./ui/resources/赛博射击屋.png)}
.QLabel:hover{background:url(./ui/resources/赛博射击屋_hover.png)}</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
